user_docs/en/userGuide.t2t

```diff
@@ -2840,6 +2840,13 @@ If the add-on was previously "disabled", the status will show "enabled after res
 If the add-on was previously "enabled", the status will show "disabled after restart".
 Just like when you install or remove add-ons, you need to restart NVDA in order for changes to take effect.
 
++++ Reviewing and consulting feedback for add-ons +++[AddonStoreReviews]
+Before installing an add-on, you may want to see opinions and reactions by others.
+Also, you may want to provide feedback about add-ons to help other users.
+To see or share your feedback about add-ons, select an add-on and use the See or provide feedback option in the actions menu.
+This will open a dedicated webpage at GitHub Discussions, where you will be able to see or share your reactions for the add-on, or even for specific versions.

```

Comment:
Again, you probably aren't going to go there to see your own reactions, especially if you haven't written them yet. I played with the language a bit in this one.

```suggestion
This will open a dedicated webpage at GitHub Discussions, where you will be able to read and write reviews for the add-on, or even comment on specific versions.
```